bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ChannelItemProvider.java

```diff
@@ -55,10 +57,12 @@
     private Map<String, Item> items = null;
 
     private boolean enabled = true;
+    private boolean initialized = false;
+    private long lastUpdate = new Date().getTime();
```

Comment:
I would prefer to use `System.nanoTime()` because this one should be really a monotonic time base.